ci/Jenkinsfile.release

```diff
@@ -9,96 +9,27 @@ def wait_for_db() {
     }
 }
 
-stage('Build Schema') {
-    node('docker') {
-        node_info()
-
-        checkout scm
-        def rustImage = docker.build('permaplant-rust', './ci/container-images/permaplant-rust')
-
-        docker.image('postgis/postgis:13-3.1').withRun('-e "POSTGRES_USER=ci" -e "POSTGRES_PASSWORD=ci"') { c ->
-            rustImage.inside("--link ${c.id}:db -e 'DATABASE_URL=postgres://ci:ci@db/ci'") {
-                checkout scm
-
-                wait_for_db()
-
-                sh './ci/build-scripts/build-schema.sh'
+// Imports
+node('docker') {
+    checkout scm
+    common = load('ci/Jenkinsfile.groovy')
```

Comment:
Will this work? There isn't even such a file?

Reply:
Good point.